sgpt/client.py

```diff
@@ -58,7 +59,15 @@ def _request(
             timeout=REQUEST_TIMEOUT,
             stream=stream,
         )
-        response.raise_for_status()
+        try:
```

Comment:
There is no reason to use `try` block here, we can just check `response.status` above.